pkg/eosclient/eosclient.go

```diff
@@ -27,7 +27,7 @@ import (
 
 // EOSClient is the interface which enables access to EOS instances through various interfaces.
 type EOSClient interface {
-	AddACL(ctx context.Context, auth, rootAuth Authorization, path string, a *acl.Entry) error
+	AddACL(ctx context.Context, auth, rootAuth Authorization, path string, position uint, a *acl.Entry) error
```

Comment:
We should add the position parameter to `UpdateACL` as well

Reply:
@ishank011 I've added the parameter, in the future we may consider to remove the UpdateACL call as it not used.